### PR TITLE
interfaces: Add /run/uuid/request to openvswitch

### PR DESCRIPTION
**Pull request by coreycb**
The openvswitch-switch deb depends on uuid-runtime, which contains
the uuid daemon. libuuid connects to the daemon via a unix-domain
socket named /run/uuidd/request.

### CODE REVIEW

**Comment by coreycb** [comment]
Thanks @mvo5. That is a great question. I'm not really sure what the advantage is. libuuid is described as generating uuids that may be accessible beyond the local system, and generating uuids that are unique within a system as well as across all systems. So perhaps that's the advantage.

**Comment by mvo5** [comment]
@coreycb Thanks. This can go in once we have a review from security (I added Jamie to the reviewers).

**Comment by jdstrand** [comment]
Based on the description, I think this belongs in the openvswitch-support interface, not the openvswitch interface. The openvswitch-support interface is meant to be used by the openvswitch daemon itself, whereas the openvswitch interface is for communicating with the openvswitch daemon via its socket. Put another way, why is libuuid needed by a client of openvswitch to communicate with the openvswitch daemon?

**Comment by coreycb** [comment]
I agree. I'll update openvswitch-support and resubmit.

File: interfaces/builtin/openvswitch_support.go [hunk 1]
```diff
@@ -29,16 +29,21 @@ const openvswitchSupportBaseDeclarationSlots = `
...
+/run/uuidd/request rw,
+`
+
 var openvswitchSupportConnectedPlugKmod = []string{`openvswitch`}
 
 func init() {
 	registerIface(&commonInterface{
 		name:                     "openvswitch-support",
-		summary:                  openvswitchSummary,
+		summary:                  openvswitchSupportSummary,
```
Comment (zyga) [inline interfaces/builtin/openvswitch_support.go]: Thanks!

**Pull request merged**